packages/graphql/tests/integration/directives/fulltext.int.test.ts

```diff
@@ -74,672 +95,2324 @@ describe("@fulltext directive", () => {
 
         await driver.close();
     });
-
-    test("should create index if it doesn't exist and then query using the index", async () => {
+    describe("Functionality", () => {
```

Comment:
:thinking: This feels like a too generic description to be really useful.

Consider wether we can find a better name, or maybe this should not even be a `describe` block

Reply:
The describe block is mainly being used for efficiency so that the before each is run only for tests that need it. I've renamed this to Query tests so hopefully that should be less generic